pkg/reconciler/revision/resources/deploy.go

```diff
@@ -90,7 +91,13 @@ func rewriteUserProbe(p *corev1.Probe, userPort int) {
 	}
 }
 
-func makePodSpec(rev *v1.Revision, loggingConfig *logging.Config, tracingConfig *tracingconfig.Config, observabilityConfig *metrics.ObservabilityConfig, deploymentConfig *deployment.Config) (*corev1.PodSpec, error) {
+func makePodSpec(rev *v1.Revision,
+	features *config.Features,
+	loggingConfig *logging.Config,
+	tracingConfig *tracingconfig.Config,
+	observabilityConfig *metrics.ObservabilityConfig,
+	deploymentConfig *deployment.Config) (*corev1.PodSpec, error) {
```

Comment:
Should we pass through the store or context instead? 😅

Reply:
separate PR?